jsk_rviz_plugins/src/overlay_image_display.cpp

```diff
@@ -149,7 +149,6 @@ namespace jsk_rviz_plugins
   void OverlayImageDisplay::processMessage(
     const sensor_msgs::Image::ConstPtr& msg)
   {
-    boost::mutex::scoped_lock(mutex_);
```

Comment:
do we need many `boost::mutex::scoped_lock lock(mutex_);` in `OverlayImageDisplay::update~~()`?
I'm not familiar with rviz plugins.